Sources/protoc-gen-swift/GeneratorOptions.swift

```diff
@@ -140,13 +163,22 @@ class GeneratorOptions {
       visibilitySourceSnippet = "package "
     }
 
-    self.implementationOnlyImports = implementationOnlyImports
     self.experimentalStripNonfunctionalCodegen = experimentalStripNonfunctionalCodegen
 
+    self.importDirective = switch (implementationOnlyImports, useAccessLevelOnImports) {
```

Comment:
CI caught that this is too new for some of the toolchains we support.  You'll have to write it slightly differently.